models/intermediate/int_quickbooks__invoice_join.sql

```diff
@@ -21,6 +21,7 @@ estimates as (
 ),
 {% endif %}
 
+{% if var('using_payment', True) %}
```

Comment:
The `int_quickbooks__invoice_join` model is only used as an upstream model in the `quickbooks__ap_ar_enhanced` model.

![image](https://github.com/fivetran/dbt_quickbooks/assets/74217849/bd7efc2a-0f70-4248-883f-f3c57d03561f)

Additionally, this model is not materialized and is [ephemeral in our config](https://github.com/fivetran/dbt_quickbooks/blob/9f759070707abe0971ee7199335abca61b391a7c/dbt_project.yml#L17-L18). 

Because of this, would it make more sense to take the same approach you took for the `quickbooks__ap_ar_enhanced` model and just entirely disable this model if payments are disabled? This would allow us to reduce the code changes and not run code in the customers warehouse if it is not entirely necessary.

Reply:
Ah I think this is a good point-- There's no point in running `int_quickbooks__invoice_join` if the downstream `quickbooks__ap_ar_enhanced` is disabled. I added the same configs (invoice and payments enabled) to this model.

Reply:
Looks great! Technically, you can revert your changes from before where the specific `{% if var('using_payment', True) %}` conditionals are included. Now that this model won't be run there is no need for these added conditionals.

Reply:
You can safely use the previous code from this model, but only make the following change and that should return the desired results
```sql
{{ config(enabled=var('using_invoice') and var('using_payment', True)) }}
```

Reply:
ah good point-- updated to revert and remove those original changes